engines/native/sandbox.go

```diff
@@ -125,6 +137,21 @@ func fetchContext(context, destdir string) error {
 	if err = os.Chmod(filename, 0700); err != nil {
 		return fmt.Errorf("Error setting file '%s' permissions: %v", filename, err)
 	}
+	u, err := osuser.Lookup(user.Name())
```

Comment:
@walac shouldn't make these things part of the `system` package... I doubt this will work well across platforms...